src/auth/auth.ts

```diff
@@ -127,6 +129,18 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> {
   protected readonly idTokenVerifier: FirebaseTokenVerifier;
   protected readonly sessionCookieVerifier: FirebaseTokenVerifier;
 
+  /**
+   * When true the SDK should communicate with the Auth Emulator for all API
+   * calls and also produce unsigned tokens.
+   * 
+   * This alone does <b>NOT<b> short-circuit ID Token verification. 
+   * For security reasons that must be explicitly disabled through
+   * setIdTokenVerificationEnabled(false);
+   */
+  protected static useEmulator(): boolean {
```

Comment:
I suggest we implement this as a module-level function (unexported). No reason why it should be in this class.  Since the `Auth` object is exposed to the developer, let's only add the bare minimum to it.